copier/cli.py

```diff
@@ -71,7 +71,7 @@ def handle_exceptions(method, *args, **kwargs):
         return 1
     except UnsafeTemplateError as error:
         print(colors.red | "\n".join(error.args), file=sys.stderr)
-        return 2
+        return 1 << 2
```

Comment:
I so rarely shift bits that without knowing it's 4, I would have had to look up on the web. Is that how the other exit codes are written within the code base? If not, any reason why not using `0b100` which, to me at least, is more readable? In comparison, `1 << 2` adds an indirection: `1` shifted by 2, so `100`, then in decimal gives `4`.

Also, maybe dumb question, but why not using `4` directly, with a short comment about using only multiples of 2?

Reply:
Fair point. I've replaced the bitwise shift by a literal `4` with a comment and in the tests just with a literal `4`.